Updating wording for contacting support (#1045)
codatio/codat-docs: docs/integrations/accounting/xero/xero-faq.md

If you need to increase or decrease this date range, edit the value of the `syncFromUTC` property for the `bankTransactions` data type in your additional sync settings (via a request to `POST /companies/{companyId}/syncSettings`).

> You can set `syncFromUTC` for all companies or for individual companies. For more information, see <a  class="external" href="https://codat.zendesk.com/hc/en-gb/articles/360018829477-Additional-sync-settings" target="_blank">Additional sync settings</a> in the Codat Knowledgebase or raise a ticket with our support team through [this form](https://codat.zendesk.com/hc/en-gb/requests/new) for further assistance.

Reviewer: ```suggestion
You can set `syncFromUTC` for all companies or individual companies. For more information, see [Advanced sync settings](/knowledge-base/advanced-sync-settings) or raise a ticket with our support team through our [support request form](https://codat.zendesk.com/hc/en-gb/requests/new).
```
